inference/scripts/build-wasm.py

```diff
@@ -131,7 +128,7 @@ def revert_git_patch(repo_path, patch_path):
     subprocess.check_call(["git", "apply", "-R", "--reject", patch_path], cwd=PROJECT_ROOT_PATH)
 
 
-def build_bergamot(args: ArgNamespace):
+def build_bergamot(args):
```

Comment:
Question: Did you mean to remove the type here? Does it not pull in correctly? Generally I try to type things at the function boundaries.

Reply:
I did remove it intentionally. The explicit import of `namedtuple` struck me as odd because I grepped for it within this repo and didn't see any instances of it. 

Given that some of this script was ported over from mozilla-unified, and how `argparse` itself seems to be a well-understood staple within the Python ecosystem, I decided to take it out. 

It looks like most of the scripts here use `Optional[list[str]] = None`, which I neglected to add. 

I'm happy to put the `namedtuple` back if you want it there. 

I think I would take the automatically provided `--help` menu as a source of truth for the script args.

```
❯ ./inference/scripts/build-wasm.py --help
usage: build-wasm.py [-h] [--clobber] [--debug] [-j J]

options:
  -h, --help  show this help message and exit
  --clobber   Clobber the build artifacts
  --debug     Build with debug symbols, useful for profiling
  -j J        Number of cores to use for building (default: all available cores)
```

Reply:
Ah yeah this is a bit weird. I don't see it used anywhere else, so this is fine.